docs/configuration.md

```diff
@@ -52,6 +52,10 @@ openstack floating ip create <public network>
 ```
 The IP must be exposed as an environment variable `OPENSTACK_CONTROLPLANE_IP`.
 
+Note: If you encounter `rule:create_floatingip and rule:create_floatingip:floating_ip_address` when create floating ip, check
```

Comment:
Is there something missing here? :)


```suggestion
Note: If you encounter errors when create floating ip, check
```

Reply:
Should we mention that an alternative is creating the floating ip before? (like @hidekazuna described in his PR?)

Reply:
I think this kind of note including concrete error message  should be added to troubleshooting.md.

General note is added in #576 as `Note: Only user with admin role can create a floating IP with specific IP.`
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/576/files#diff-76e731333fb756df3bff5ddb3b731c46R58.